dist/mk-docker-opts.sh

```diff
@@ -11,7 +11,7 @@ OPTIONS:
 	-c	Output combined Docker options into DOCKER_OPTS var
 	-k	Set the combined options key to this value (default DOCKER_OPTS=)
 	-m	Do not output --ip-masq (useful for older Docker version)
-" >/dev/stderr 
```

Comment:
I think /dev/stderr is a bashism and >&2 is more POSIX compliant. See https://lists.debian.org/debian-user/2009/07/msg00014.htmlfor some discussion on the subject.